src/messages/MClientRequest.h

```diff
@@ -212,6 +221,8 @@ class MClientRequest final : public MMDSOp {
       decode(stamp, p);
     if (header.version >= 4) // epoch 3 was for a ceph_mds_request_args change
       decode(gid_list, p);
+    if (header.version >= 5)
+      decode(alternate_name, p);
```

Comment:
Interestingly, the kernel is still on version 2 of this struct. I wonder if we ought to plumb in support for the some of the other fields first. The gid_list in particular...

Reply:
Okay, I didn't notice this, I will check it and let me add it if needed.

Reply:
That's really a separate issue. You shouldn't need to worry about it as part of this PR. I'll open a tracker for it.

Reply:
Patches posted today to ceph-devel add support for gid_list. That set should clear the way for us to add alternate_name support.

Reply:
Cool, I will test your patches today. Thanks.